react-image-annotate/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "react-image-annotate",
```

Comment:
We may want to customise some of these parameters to indicate that we have made changes from the base library.